### ISSUE TITLE
on.cypress.io/test-retries redirect does not work: 404

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress-documentation#3824]
https://on.cypress.io/test-retries does not go anywhere

<img width="1012" alt="Screen Shot 2021-04-11 at 11 57 43 AM" src="https://user-images.githubusercontent.com/2212006/114311544-32666580-9abd-11eb-9c45-d2392dbc766f.png">

**Comment by Abhijith2893** [comment]
Since there is a 'guides' sub menu within the 'guides' tab, the url is `https://docs.cypress.io/guides/guides/test-retries`

**Comment by matthamil** [comment]
I don't think this is actionable in this repo since the `on` service links are not handled here.

**Comment by the0neWhoKnocks** [comment]
Ran into a similar issue. While on the doc site I did a search (in the top right Search input) for `read` which returned a list that had a `readFile` entry (what I was looking for), so I clicked on it. For a couple seconds I was shown the correct page, then it suddenly changes to a 404 page.

If I then go to Google, and look up `cypress readfile`, it gives me a result for the same doc page, I click on it, and the page doesn't eventually change to a 404.

After comparing the URLs, there appears to be a casing issue with your resolution code.
```
# Google URL
https://docs.cypress.io/api/commands/readfile

# Doc Search URL
https://docs.cypress.io/api/commands/readFile
```
So `readfile` (good) vs `readFile` (bad)

**Comment by emilyrohrbough** [comment]
This redirect appear to be working at this time.
